src/pages/MyKiva/MyKivaPageContent.vue

```diff
@@ -83,26 +83,85 @@
 				<h3
 					class="tw-text-center tw-mb-2"
 				>
-					Your Achievements
+					<span
+						class="tw-bg-secondary tw-text-primary tw-px-1 tw-text-h4"
+						style="line-height: 0; font-weight: 600;"
+					>
+						Your Achievements
```

Comment:
Also revert the changes on this title to match the work from the ticket linked in the other comment.